hawkular-bus-common/src/main/java/org/hawkular/bus/common/consumer/RPCBasicMessageListener.java

```diff
@@ -127,14 +130,6 @@ public void onMessage(Message message) {
         } catch (Exception e) {
             msglog.errorFailedToSendResponse(e);
             return;
-        } finally {
```

Comment:
why is the finally removed? should we not close the consumer connection context? We could do that session close in here as well.

Also - if not needed anymore, remove the errorFailedToCloseResourcesToRPCClient from the MsgLogger interface.